valhalla/sif/edgelabel.h

```diff
@@ -364,20 +359,20 @@ class EdgeLabel {
    * deadend_:        Flag indicating edge is a dead-end.
    * on_complex_rest: Part of a complex restriction.
    */
-  uint64_t endnode_          : 48;   // Could be 46 (2 spare)
-  uint64_t use_              : 6;
-  uint64_t classification_   : 3;
-  uint64_t shortcut_         : 1;
-  uint64_t dest_only_        : 1;
-  uint64_t origin_           : 1;
-  uint64_t toll_             : 1;
-  uint64_t not_thru_         : 1;
-  uint64_t deadend_          : 1;
-  uint64_t on_complex_rest_  : 1;
-
-  Cost cost_;       // Cost and elapsed time along the path.
-  float sortcost_;  // Sort cost - includes A* heuristic.
-  float distance_;  // Distance to the destination.
+  uint64_t endnode_ : 48; // Could be 46 (2 spare)
```

Comment:
I like the way the bitfield sizes were lined up so they are easy to add. Guess I can get over this change

Reply:
Hmm, I agree this is unfortunate.  There doesn't seem to be a styling rule available to configure this.

There's one ticket that's sort of related: https://bugs.llvm.org/show_bug.cgi?id=19067

It makes the point that one problem that arises with this kind of alignment is that introduction of a new long variable in the list can change minimum column width, and require re-formatting all variables in the block to re-align them, adding formatting noise to each diff unrelated to the change.

@dnesbitt61 We could disable formatting on these sections to keep the alignment - I do kind of like it to.

http://clang.llvm.org/docs/ClangFormatStyleOptions.html#disabling-formatting-on-a-piece-of-code

Do you feel strongly enough about it for me to update these blocks?

Reply:
No. actually the comment about a long name requiring a whole block to be changed to re-align makes me favor the new way. I keep forgetting how many times I have had to realign things. Of course it sort of makes me feel good to "keep order" in the code but it definitely wastes time.

Reply:
👍 I think we're ready to merge then!